flow2ml/Flow.py

```diff
@@ -202,3 +204,23 @@ def getDataset(self,img_dimensions,train_val_split):
     self.update_progress( 100/100.0,"Created Datasets" ) 
 
     return self.dataset
+
+  def deployTensorflowModels(self,model):
+    '''Deploy conversion from tensorflow model to tensorflowjs or tflite model'''
+
+    if(self.conversions['tfjs']==self.conversions['tflite']):
```

Comment:
You won’t have conversions in self
Please change this to conversions instead of self.conversions in all places